Segundo Pull Request

rodrigo-arnaiz opened the pull request:
Link de vercel: http://beervana-4fkveztm0-rodrigos-projects-fa5acc60.vercel.app/

Se añadieron estilos al panel de admin, un login, y el listado de cervezas. Faltan las views de los abm, una base de datos, un superusuario(creado mediante un seeder) y la api para las ventas en un futuro.

Matiaseze committed:
credenciales de ejemplo para un mysql en docker

LucianApzz committed:
login hecho

Matiaseze committed:
Dashboard con template base para admin-panel. Listado de marcas y cervezas añadido. Alta de marca añadida

Matiaseze committed:
Cambios en rutas (temporal) para que la pagina por defecto sea el dashboard. Home requiere auth para probar el login

Matiaseze committed:
modificacion en conf de vercel para incluir la carpeta public/assets de imagenes

Matiaseze committed:
Cambio en la config de la base de datos para conectarse con supabase

hgabrieltw reviewed (approved):
Aunque el acceso a los datos de la base da error en la consulta se notan los avances. Traten de mostrar algo más concreto de los abm.

LucianApzz committed:
ABM completos de cervezas/marcas/estilos/fermentacion + migración detalle tipo_fermentacion
- Controllers actualizados
- Vistas hechas
- Sidebar y rutas ajustadas
- Imagen de muestra cerveza.jpeg

rodrigo-arnaiz committed:
Se agregaron tablas como factura y detalle factura, hay un seeder para un admin y un usuario normal, login funcional, se acomodo la pagina principal y el sidebar, y se agrego una configuracion inicial para la api y sanctum

rodrigo-arnaiz committed:
Se arreglo el modificar cerveza para cambiar la imagen

rodrigo-arnaiz committed:
excusa para tener un nuevo deploy

rodrigo-arnaiz committed:
excusa para el deploy

rodrigo-arnaiz committed:
cambio php vercel.json

rodrigo-arnaiz committed:
se usa la ultima version de php

rodrigo-arnaiz committed:
correcion de funcion sin propiedades

rodrigo-arnaiz committed:
mas cambios

rodrigo-arnaiz committed:
mas cambios

rodrigo-arnaiz committed:
cambios para usar node 18
